graphql_compiler/compiler/blocks.py

```diff
@@ -243,6 +244,10 @@ def validate(self):
 
         validate_safe_string(self.edge_name)
 
+    def get_field_name(self):
+        """Return the field name corresponding to the edge being traversed."""
+        return u'{}_{}'.format(self.direction, self.edge_name)
```

Comment:
@jssandh2 named parameters are slightly nicer because it's harder to flip the order by accident, and it's more obvious when it happens. This is probably fine too, but if someone has named them already then let's leave it be in the future.